qt5/build.sh

```diff
@@ -1,96 +1,93 @@
 #!/bin/bash
 
-BIN=$PREFIX/lib/qt5/bin
-QTCONF=$BIN/qt.conf
-
-chmod +x configure
+# change qt.conf name for qt4 coexistence
+patch -p1 <$RECIPE_DIR/qt5_conf.patch
```

Comment:
I mean, after Qt4 packages are released based on the recipes present in #616, Qt5 packages based on our current recipes could be installed side by side with them, so this patch is not needed.